<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>642</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>741</width>
      <height>492</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_14" stretch="1,1">
       <item>
        <widget class="QComboBox" name="chooseModeComboBox">
         <property name="minimumSize">
          <size>
           <width>76</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="1,1">
       <item>
        <widget class="QStackedWidget" name="stackedWidget">
         <property name="layoutDirection">
          <enum>Qt::LayoutDirection::LeftToRight</enum>
         </property>
         <property name="lineWidth">
          <number>1</number>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="page">
          <widget class="QPushButton" name="readMatrButton">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>1</y>
             <width>371</width>
             <height>26</height>
            </rect>
           </property>
           <property name="text">
            <string>Load matrices from a file</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="page_2">
          <widget class="QPushButton" name="inputMatrButton">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>1</y>
             <width>371</width>
             <height>26</height>
            </rect>
           </property>
           <property name="text">
            <string>Manually input matrix</string>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="solveMatrButton">
         <property name="text">
          <string>Solve matrix chain multiplication</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="parenthesizationLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelResult">
       <property name="styleSheet">
        <string notr="true">QLabel { color: rgb(0, 0, 127); }</string>
       </property>
       <property name="text">
        <string>Final matrix result</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="resultMatrTable"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="labelValue">
         <property name="styleSheet">
          <string notr="true">QLabel { color: rgb(0, 0, 127); }</string>
         </property>
         <property name="text">
          <string>Optimal value</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelSolutionOptimal">
         <property name="styleSheet">
          <string notr="true">QLabel { color: rgb(0, 0, 127); }</string>
         </property>
         <property name="text">
          <string>Optimal solution reconstruction</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QTableWidget" name="costTable"/>
       </item>
       <item>
        <widget class="QTableWidget" name="orderTable"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QDialogButtonBox" name="buttonBox">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>520</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton[text=&quot;OK&quot;] {
    color:  rgb(0, 0, 127); ;
}

QPushButton[text=&quot;Cancel&quot;] {
    color: rgb(199, 0, 0);
}</string>
    </property>
    <property name="standardButtons">
     <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
